### PR TITLE
fix: Do not add load() method to generated PO interface file

### PR DESCRIPTION
**Pull request by jimevanssfdc**

### CODE REVIEW

File: utam-compiler/src/test/java/utam/compiler/translator/InterfaceSerializerTests.java [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+public class InterfaceSerializerTests {
+
+  private static String getImplementationCode(String json) {
+    PageObjectInterface pageObject = getJsonStringDeserializer(json).getObject().getInterface();
+    return new InterfaceSerializer(pageObject).toString();
+  }
+
+  @Test
+  public void testToString() {
+    String json =
```
Comment (AdamTReineke) [inline utam-compiler/src/test/java/utam/compiler/translator/InterfaceSerializerTests.java:20]: This isn't the minimal test for the change and the test name isn't meaningful. If this fails, how would someone know what was supposed to be happening?

File: utam-compiler/src/main/java/utam/compiler/translator/InterfaceSerializer.java [hunk 2]
```diff
@@ -53,6 +54,12 @@ private String getPackageName() {
   }
 
   private void addMethodDeclaration(List<String> out, MethodDeclaration declaration) {
+    // The interface extends the PageObject interface, which already has a load
+    // method, so we don't need to add a load method to the generated interface.
+    if (declaration.getName().equals(BEFORE_LOAD_METHOD_NAME)) {
```
Comment (AdamTReineke) [inline utam-compiler/src/main/java/utam/compiler/translator/InterfaceSerializer.java:59]: This should probably verify that `source.getBaseInterfaceType().getSimpleName()` is the interface the `PageObject` interface too.

**Review by AdamTReineke — APPROVED** [review]

File: utam-compiler/src/test/java/utam/compiler/translator/InterfaceSerializerTests.java (hunk 1 above)
Comment (jimevans) [reply utam-compiler/src/test/java/utam/compiler/translator/InterfaceSerializerTests.java:20]: The extant test will be blatantly obvious what the issue is. We can undertake a test refactoring later if we really feel strongly about this.

File: utam-compiler/src/main/java/utam/compiler/translator/InterfaceSerializer.java (hunk 2 above)
Comment (jimevans) [reply utam-compiler/src/main/java/utam/compiler/translator/InterfaceSerializer.java:59]: It _could_ verify that, but the verification would be superfluous, as the only thing this class will ever serialize is an interface extending `PageObject`. If that ever changes, yes, we'd probably want to make a change here.

**Pull request merged**